activities/github-doc-workflow.md

```diff
@@ -0,0 +1,98 @@
+# GitHub workflow for pull requests
+
+For some changes, it can be simpler and quicker to submit a pull request using
+only GitHub rather than using git and your local development
+environment. In this section, we'll edit the documentation for a repository
+that you do not have cloned locally (GMT) on GitHub.
+
+We'll tackle [issue 5653](https://github.com/GenericMappingTools/gmt/issues/5653)
+in the GMT repository while learning how to submit pull requests on GitHub.
+The workshop leaders will assign each participant one of the files to change.
+We'll complete the steps together and you can reference the instructions below.
+
+## Creating a pull request by editing documentation on GitHub
+
+- Go to [GMT issue 5653 on GitHub](https://github.com/GenericMappingTools/gmt/issues/5653).
+- Click on the module link on the section that you have been assigned to go to
```

Comment:
Perhaps better to use a numbered list (1, 2, 3) instead of bullets for this file?

```suggestion
1. Go to [GMT issue 5653 on GitHub](https://github.com/GenericMappingTools/gmt/issues/5653).
2. Click on the module link on the section that you have been assigned to go to
```

Reply:
OK, I'll update the rest.